client/app/scripts/components/help-panel.js

```diff
@@ -27,20 +35,135 @@ function renderShortcuts(cuts) {
   );
 }
 
-export default class HelpPanel extends React.Component {
-  render() {
-    return (
-      <div className="help-panel">
+
+function renderShortcutPanel() {
+  return (
+    <div className="help-panel-shortcuts">
+      <h2>Shortcuts</h2>
+      <h3>General</h3>
+      {renderShortcuts(GENERAL_SHORTCUTS)}
+      <h3>Canvas Metrics</h3>
+      {renderShortcuts(CANVAS_METRIC_SHORTCUTS)}
+    </div>
+  );
+}
+
+
+const BASIC_SEARCHES = [
+  {term: 'foo', label: 'All fields for foo'},
+  {term: 'pid: 12345', label: 'Any field matching pid for the value 12345'},
+];
+
+
+const REGEX_SEARCHES = [
+  {term: 'foo|bar', label: 'All fields for foo or bar'},
+  {term: 'command: foo(bar|baz)', label: 'Command field for foobar or foobaz'},
+];
+
+
+const METRIC_SEARCHES = [
+  {term: 'cpu > 4%', label: 'CPU greater than 4%'},
+  {term: 'memory < 10mb', label: 'memory less than 4mb'},
```

Comment:
this should be `...than 10MB`, capital M is for Mega, and capital B is for byte, as opposed to lowercase b (bit).